projects/innov2017/generate_stack.py

```diff
@@ -111,11 +126,16 @@ def git_commit(self):
             self._git_commit = commit
         return self._git_commit
 
+    def init(self):
+        self.stack = dict(version='3.6', services={}, volumes={})
```

Comment:
Please use the Pyhon 3 syntax (`self.stack = {...}`). You mixed `dict()` with `{}` and that is really confusing.

Reply:
Good point, I will fix